CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Add to rootfs build for wlconf [Minh]
```

Comment:
Maybe change this to:

"Add wlconf to rootfs [Minh]"

Reply:
So I have to create another PR or just add commit?